matlab/@amimodel/parseModel.m

```diff
@@ -184,7 +184,7 @@ function parseModel(this)
 if(strcmp(this.wtype,'iw'))
     if(isfield(this.fun,'M'))
         this.getFun([],'M');
-        this.id = double(logical(sum(this.fun.M.sym,2)~=0));
+        this.id = double(logical(any(this.fun.M.sym)));
```

Comment:
Shouldn't it be `any(this.fun.M.sym, 2)` ?

Reply:
That would yield the same result, not? (With `any` potentially being a bit faster - not that it really matters here)

Reply:
From my understanding, checking for differential *components* can be done by checking whether any of the respective columns of the mass matrix are nonzero, to see whether the time derivative of the respective component appears *anywhere* in the rhs.

Reply:
Ah, okay. I see my error. Indeed, your code is fine!